test/integration/outreach/outreach.js

```diff
@@ -33,8 +33,16 @@ context('Outreach', function() {
       .type('Patient');
 
     cy
-      .get('.js-birth-date')
-      .type('1990-10-01');
+      .get('.js-dob-month')
+      .select('10');
```

Comment:
maybe select a single digit just to make sure we're getting the leading zeros

Reply:
Good idea, updated the test to select a single digit for both month and day.